### PR TITLE
Update documentation for Vulkan SC

### PR DESCRIPTION
**Pull request by dgkoch**

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -59,7 +62,7 @@ in this repository for more details.
 
 ## Repository Content
 
-This repository contains the source code necessary to build the Vulkan
+This repository contains the source code necessary to build the Vulkan SC
```
Comment (sricker-nvidia) [inline BUILD.md:65]: Should this read, "necessary to build the Vulkan and Vulkan SC validation layers"?

File: BUILD.md [hunk 2]
```diff
@@ -78,15 +81,15 @@ directory.
 
 ### Display Drivers
 
-This repository does not contain a Vulkan-capable driver. You will need to
-obtain and install a Vulkan driver from your graphics hardware vendor or from
+This repository does not contain a Vulkan SC-capable driver. You will need to
```
Comment (sricker-nvidia) [inline BUILD.md:84]: Does not contain a Vulkan or Vulkan SC ... ?

File: BUILD.md (hunk 1 above)
Comment (dgkoch) [reply BUILD.md]: sure. Done.

File: README.md [hunk 3]
```diff
@@ -22,9 +22,26 @@ The following components are available in this repository:
...
+and is not expected to be used in **production** (safety) environments.
+
+**NOTE**: This repository is a downstream fork of [KhronosGroup/Vulkan-ValidationLayers](https://github.com/KhronosGroup/Vulkan-ValidationLayers)
+which is periodically rebased.
+
+The `sc_main` branch is currently based on the `sdk-1.2.198` Vulkan-ValidationLayers tag and contains Vulkan SC modifications to support the Vulkan SC 1.0 API.
+
+
 ## Contact Information
 * [Mark Lobodzinski](mailto:mark@lunarg.com)
```
[the remark this answers is not in the document]
Comment (dgkoch) [reply README.md]: done.

**Review by sricker-nvidia — APPROVED** [review]

**Pull request merged**